src/pages/AddressInfoPage/NFTList.tsx

```diff
@@ -0,0 +1,202 @@
+
+The library is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License
+along with the library. If not, see <http://www.gnu.org/licenses/>.
+*/
+
+import { motion, useMotionValue, useSpring, useTransform } from 'framer-motion'
+import { useState } from 'react'
+import styled from 'styled-components'
+
+import Card3D, { card3DHoverTransition } from '@/components/Cards/Card3D'
+import SkeletonLoader from '@/components/SkeletonLoader'
+import { deviceBreakPoints } from '@/styles/globalStyles'
+import { UnverifiedNFTMetadataWithFile } from '@/types/assets'
+
+interface NFTListProps {
+  nfts: UnverifiedNFTMetadataWithFile[]
+  isLoading?: boolean
+}
+
+const NFTList = ({ nfts, isLoading }: NFTListProps) => (
+  <NFTListStyled>
+    {nfts.map((nft) => (
+      <NFTItem key={nft.id} nft={nft} />
+    ))}
+    {isLoading && (
+      <>
+        <SkeletonLoader height="100%" />
+        <SkeletonLoader height="100%" />
+      </>
+    )}
+  </NFTListStyled>
+)
+
+interface NFTItemProps {
+  nft: UnverifiedNFTMetadataWithFile
+}
+
+const NFTItem = ({ nft }: NFTItemProps) => {
+  const [isHovered, setIsHovered] = useState(false)
+
+  const desc = nft.file?.description
+  const cutDesc = desc && desc?.length > 200 ? nft.file?.description?.substring(0, 200) + '...' : desc
+
+  const y = useMotionValue(0.5)
+  const x = useMotionValue(0.5)
+
+  const springConfig = { damping: 10, stiffness: 100 }
+  const xSpring = useSpring(x, springConfig)
+  const ySpring = useSpring(y, springConfig)
+
+  const imagePosX = useTransform(xSpring, [0, 1], ['5px', '-5px'], {
+    clamp: true
+  })
+  const imagePosY = useTransform(ySpring, [0, 1], ['5px', '-5px'], {
+    clamp: true
+  })
+
+  const handlePointerMove = (pointerX: number, pointerY: number) => {
+    x.set(pointerX, true)
+    y.set(pointerY, true)
+  }
+
+  return (
+    <NFTCardStyled
+      onPointerMove={handlePointerMove}
+      onCardHover={setIsHovered}
+      frontFace={
+        <FrontFace>
+          <NFTPictureContainer>
+            <PictureContainerShadow animate={{ opacity: isHovered ? 1 : 0 }} />
+            <NFTPicture
+              style={{
+                backgroundImage: `url(${nft.file?.image})`,
+                x: imagePosX,
+                y: imagePosY,
+                scale: 1.5
+              }}
+              animate={{
+                scale: isHovered ? 1 : 1.5
+              }}
+              transition={card3DHoverTransition}
+            />
+          </NFTPictureContainer>
+          <NFTName>{nft.file?.name}</NFTName>
+        </FrontFace>
+      }
+      backFace={
+        <BackFace>
+          <NFTDescription>{cutDesc}</NFTDescription>
+          <BackFaceBackground style={{ backgroundImage: `url(${nft.file?.image})` }} />
```

Comment:
If you just swap the order of these 2 components, then you can remove the `z-index` from both of the components.